deck.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'flashcard'
+require_relative 'card_parser'
+
+class Deck
+  attr_accessor :deck, :flashcards
+
+  include Parsable
+
+  def initialize(flashcards)
+    @flashcards = flashcards
```

Comment:
Good. I would default this to an empty array if nothing is passed. `def initialize(flashcards = [])`